docs/source/geopm_prof.3.rst

```diff
@@ -147,6 +139,17 @@ initialized.
   progress.  This method signals the completion of one work unit out
   of the total passed to ``geopm_tprof_init()``.
 
+OMPT Integration
+----------------
+
+Integrating with OMPT provides automation for region identification, entry,
+and exit. With OMPT support, a geopm report can provide per-region metrics
+(such as region runtime, CPU/GPU frequency, power/energy consumption, etc,
+without the need to mark up the application. However, for each region where
+region progress is desired, the application must be explicitly marked up with
+``geopm_tprof_post()``.
```

Comment:
Please add a pointer to this test as an example of how to do that:
https://github.com/geopm/geopm/blob/dev/integration/test/test_progress.cpp#L28